unity/PitayaExample/NugetOutput/Unity/Editor/PitayaXcodeProjectUpdater.cs

```diff
@@ -1,67 +0,0 @@
-#if UNITY_IPHONE
-using UnityEditor;
-using UnityEngine.Assertions;
-using UnityEditor.Callbacks;
-using UnityEditor.iOS.Xcode;
-using UnityEditor.iOS.Xcode.Extensions;
-using System.IO;
-
-public class PitayaBuildPostprocessor
-{
-
-  private static void CopyDirectory(string sourcePath, string destPath)
-  {
-    Assert.IsFalse(Directory.Exists(destPath));
-    Directory.CreateDirectory(destPath);
-
-    foreach (string file in Directory.GetFiles(sourcePath))
-      File.Copy(file, Path.Combine(destPath, Path.GetFileName(file)));
-
-    foreach (string dir in Directory.GetDirectories(sourcePath))
-      CopyDirectory(dir, Path.Combine(destPath, Path.GetFileName(dir)));
-  }
-
-  private const string FRAMEWORK_ORIGIN_PATH =
-    "Assets/Pitaya/Native/iOS"; // relative to project folder
-  private const string FRAMEWORK_NUGET_ORIGIN_PATH =
-    "Libraries/com.wildlifestudios.nuget.libpitaya/Native/iOS"; // relative to build folder
-  private const string FRAMEWORK_TARGET_PATH =
-    "Frameworks"; // relative to build folder
-
-  private static string[] FRAMEWORK_NAMES = {"libpitaya.xcframework", "libcrypto.xcframework", "libssl.xcframework", "libuv.xcframework"};
-
-  [PostProcessBuild]
-  public static void OnPostprocessBuild(BuildTarget buildTarget, string path)
```

Comment:
Is this postprocessor not going to be necessary anymore? I'm asking especially because of the `-lz` flag addition to the iOS project.

Reply:
No. It was necessary when we tried to use the `xcframeworks`.
I've added [this change](https://github.com/topfreegames/libpitaya/commit/846df475c0d0b5bad5d9b63edfa7ea5d0a274e12#diff-060156af0b865f16d99ec6e3c92bff5b04112c9950061772ebf4307349c64f8a) which will copy all files in the Editor folder to the nuget package, and [the Editor](https://github.com/topfreegames/libpitaya/blob/fix/unity-example/unity/PitayaExample/Assets/Pitaya/Editor/PitayaXcodeProjectUpdater.cs) folder contains the correct script.

Reply:
Got it, LGTM, then!